content/_index.md

```diff
@@ -1,29 +1,32 @@
-LinuxBoot is the concept of using a Linux kernel and initramfs as a bootloader.
+LinuxBoot is the concept of [using a Linux kernel and initramfs as a
+bootloader](#benefits).
+
+![](images/linuxboot_info.png)
+
+The official documentation for LinuxBoot is the [LinuxBoot Book 📖](https://book.linuxboot.org),
+where you can find information on how LinuxBoot works and implementing it with
+various approaches, such as based on coreboot or UEFI PEI.
```

Comment:
"Implementing it with various approaches" would be how you configure the kernel or what you use for userspace. coreboot and UEFI PEI are two examples of loaders of LinuxBoot, not strictly  examples of implementation approaches, although they can impact the implementation. I'd also either remove the explicit examples or at least add U-Boot SPL. This is just two of the 4 examples already in `linuxboot_info.png`.